rsoxs/Functions/energyscancore.py

```diff
@@ -366,7 +366,7 @@ def new_en_scan_core(
     master_plan=None,   # if this is lying within an outer plan, that name can be stored here
     sim_mode=False,  # if true, check all inputs but do not actually run anything
     md=None,  # md to pass to the scan
-    signals = [Beamstop_WAXS_int, Beamstop_SAXS_int, Izero_Mesh_int, Sample_TEY_int,mono_en,epu_gap,ring_current],
+    signals = [Beamstop_WAXS_int, Beamstop_SAXS_int, DM7_Photodiode_int, Izero_Mesh_int, Sample_TEY_int,mono_en,epu_gap,ring_current],
```

Comment:
I don't think it's relevant to measure in an RSoXS scan?  generally the detector being in makes this not really useful, right?

Reply:
Oh, I see, I took it back out.  Maybe not in this pull request, but do you think it makes sense down the line to rename the function so it is more obvious that it is an RSoXS scan?  The function name gave me the impression that it was just a general energy scan universally applicable to RSoXS or NEXAFS.